src/src/tx_main.cpp

```diff
@@ -1006,6 +1023,11 @@ void setup()
     }
   }
 
+#if defined(HAS_BUTTON)
+  registerButtonFunction(ACTION_BIND, EnterBindingMode);
+  registerButtonFunction(ACTION_INCREASE_POWER, cyclePower);
```

Comment:
`cyclePower` didn't work for me. It definitely immediately changed the power (log message SetPower X) but the OLED doesn't update and if I was connected, it immediately would reset back to the configured max power.

Reply:
I've changed the screen behaviour so that it will always display the current power, rather than configured power if non-dynamic and current power if dynamic. Also, the * next to the power is display if its dynamic of the current power is not the configured power.
The problem of it being reset back is an interaction between the g-sensor setting it back to the configured value if the handset goes from stationary to moving, and the dynamic power setting it back to configured on line 78 in `dynpower.cpp`, which is trying to adjust for the MinPower setting when you previously go too close to the RX!